api/v1alpha1/repository_types.go

```diff
@@ -54,10 +47,9 @@ type FilterCond struct {
 	// Name of the component
 	Name string `json:"name,omitempty"`
 
-	// default is keep
-	// +kubebuilder:validation:Enum=keep;ignore
-	// +kubebuilder:default:=keep
-	Operation FilterOp `json:"operation,omitempty"`
+	// Enable filtering on the component,
+	// if false, no action will be taken on the component, in other worlds, all versions of the component will be retained.
+	Enabled bool `json:"enabled,omitempty"`
 
 	// If True, the current version will be retained even if it is deprecated.
 	Deprecated bool `json:"deprecated,omitempty"`
```

Comment:
To make the usage clearer, how about replacing `deprecated` with `keepDeprecated`?

Reply:
@dayuy pls change `deprecated` to `keepDeprecated`